### ISSUE TITLE
Bugfix release on legacy branch

### ISSUE THREAD

**Issue by bekuno** [cgeo/WhereYouGo#165]
Do we want do further thinks on this branch before release it as version 0.9.4?

My suggestions are to adopt
 [ ]  PR #25 Updated main menu links
 [ ]  PR #32 Update about pages
 [ ]  PR #60 Changelog

**Comment by Lineflyer** [comment]
AFAICS the changes of #25 are already on legacy

**Comment by Lineflyer** [comment]
Update: This is all already down on `legacy`, isn't it?

**Comment by bekuno** [comment]
Yes, with PR #183 all points above are implemented, and additional points.
And yes, on `legacy`.

**Comment by Lineflyer** [comment]
@bekuno 
You wrote in some other issue, that we could release the legacy version (from code status).

What we need (and I cannot provide or don't know):
- Build an APK with appropriate versioncode / versionname 
  I suggest to use something like "yyyy.mm.dd-legacy" like we do for c:geo in order to be consistently able to release from Google Play also later on 
- Set minsdk/maxsdk accordingly to seperate legacy from the production build

Alternatively we could just build a version from legacy and make it available to download somewhere outside of Google Play.

I can do the Google Play handling once someone manages to prepare e.g. the CI-environment to make a proper build from legacy branch (same handling as in c:geo) or at least provides a matching APK once. Sorry, but I am technically and timely not able to assist here besides that.

**Comment by Lineflyer** [comment]
I created a new job on our CI: https://ci.cgeo.org/view/WhereYouGo/job/WhereYouGo%20(legacy%20release)/

As the automatic versioning has only been created on master/release we just set the versionCode and versionName manually in build.gradle. 
I have set it to versionName=2020.06.25-legacy with versionCode=20200625 for now and successfully built an APK.

Its basically working (incl. download of cartridges) I will additionally test it on a SDK=15 device.
Afterwards I need to check whether Google Play will accept this update and targets it to minSDK=maxSDK=15.

Will report back.

**Comment by Lineflyer** [comment]
Everything seems as expected now, also Google Play pre-check shows it will accept the version and distribute to SDK=15 only.
I asked @kurly1 in #156 to load and test the version on a Garmin Monterra (=SDK15).

**Comment by bekuno** [comment]
@Lineflyer Thanks for preparing and releasing :-)

**Comment by Lineflyer** [comment]
As discussed in #156 I will wait if maybe a solution for enabling in-app download is possible

**Comment by bekuno** [comment]
@Lineflyer The legacy branch thas targetSdkVersion 28.
How about make a new legacy release with the current changes?

**Comment by Lineflyer** [comment]
mmh...the legacy version was never published as the feedback in #156 was negative when it comes to downloading cartridges.
I guess its to late for a legacy now.

If we at some point of time have a working solution for Monterra we can offer the APK for direct download.
